src/ark/utils/data_utils.py

```diff
@@ -400,6 +480,11 @@ def generate_and_save_neighborhood_cluster_masks(fovs: List[str],
             Specify what to append at the end of every pixel mask. Defaults to `''`.
     """
 
+    ccmd = CellClusterMaskData(data=neighborhood_data,
+                               fov_col=settings.FOV_ID,
+                               label_col=settings.CELL_LABEL,
+                               cluster_col=settings.KMEANS_CLUSTER)
+
```

Comment:
Same as above - can we make all of these that have `settings` as function arguments and make these as the defaults?